server/routes/clearReadNotifications.ts

```diff
@@ -1,28 +1,35 @@
 import { Request, Response, NextFunction } from 'express';
 import { factory, formatFilename } from '../../shared/logging';
-import { sequelize, DB, } from '../database';
+import { sequelize, DB } from '../database';
 
 const log = factory.getLogger(formatFilename(__filename));
 
-export default async (models: DB, req: Request, res: Response, next: NextFunction) => {
+export default async (
+  models: DB,
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
   if (!req.user) {
     return next(new Error('Not logged in'));
   }
 
-  const subscriptions = await models.Subscription.findAll({
-    where: { subscriber_id: req.user.id },
-    include: [{
-      model: models.Notification,
-      where: { is_read: true },
-      as: 'Notifications',
-    }]
-  });
+  await sequelize.query(
```

Comment:
This should be transactionalized in case it fails.

wrap it in: 
```
await sequelize.transaction(async (t) => {
...});
```
and throw into the query as the last option: `{ transaction: t}`

Reply:
RDBMS automatically wraps singular queries in transactions I believe. Is there some sequelize error handling that wrapping in a transaction provides?